src/endpoints.go

```diff
@@ -12,16 +13,30 @@ import (
 const CONTAINER_STOP_TIMEOUT = 10 // seconds
 
 var (
-	daemonPort   int    = 2375
-	daemonHost   string = fmt.Sprintf("http://127.0.0.1:%d", daemonPort)
-	dockerClient *docker.Client
+	daemonPort    int = 2375
+	daemonTLSPort int = 2376
+	dockerClient  *docker.Client
 )
 
 func init() {
-	daemonHost = fmt.Sprintf("http://%s:%d", getDaemonHostIp(), daemonPort)
+	var daemonHost string
+	var client *docker.Client
+	var err error
+	certPath := os.Getenv("DOCKER_CERT_PATH")
+	if certPath == "" {
+		daemonHost = fmt.Sprintf("tcp://%s:%d", getDaemonHostIp(), daemonPort)
+		client, err = docker.NewClient(daemonHost)
+	} else {
+		daemonHost = fmt.Sprintf("tcp://%s:%d", getDaemonHostIp(), daemonTLSPort)
+		log.Info("Docker cert path: %s", certPath)
+		ca := fmt.Sprintf("%s/ca.pem", certPath)
```

Comment:
Ah no, names matches to tlscacert, tlscert and tlskey but these envars would be specific to `dockerface`. :)
My concern is that on linux I would have to rename these files or create aliases to satisfy them.